public/app/plugins/datasource/tempo/language_provider.ts

```diff
@@ -21,19 +21,25 @@ export default class TempoLanguageProvider extends LanguageProvider {
   };
 
   start = async () => {
-    await this.fetchTags();
-    return [];
+    if (!this.startTask) {
```

Comment:
Just for context, no change needed, we don't need this to support the `LanguageProvider` interface. That interface was created with a goal of interchangeability I guess but right now there is no place where conforming to that interface is required or has any benefits. At that point this initialization may be a bit simplified.